### PR TITLE
Makes the cult less likely to accidentally cuck itself

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscdel: You can no longer scribe the Summon Nar-Sie rune under non-valid conditions.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
Invalid

**Comment by coiax** [comment]
Oh no, think of all the poor cultists trying to summon narnar when they're already here :'(

**Comment by Shadowlight213** [comment]
Hugbox. Cult should communicate if the target is sacced or not.
Alternatively make it give hell water if they haven't sacced yet.

**Comment by PKPenguin321** [comment]
So if I still remember right, cult being able to summon narsie without having sacrificed their target was intentionally left in so a cult could go last-ditch summon and get at least a half win

**Comment by ChangelingRain** [comment]
Cultists can't summon unless the target is sacced, so making the rune before the target is sacced is literally worthless.

**Comment by Shadowlight213** [comment]
True, but if the cult isn't communicating, then that's on them. Makes all the protection the summoner gets kinda overkill if there's no chance of failure.
If thus gets merged, I'll probably nerf the shields to 3x3 and like 10 hp post freeze.

File: code/game/gamemodes/cult/ritual.dm [hunk 1]
```diff
@@ -203,21 +203,35 @@ This file contains the arcane tome files.
...
+			for(var/B in spiral_range_turfs(1, user, 1))
+				var/turf/T = B
+				var/obj/machinery/shield/N = new(T)
+				N.name = "Rune-Scriber's Shield"
+				N.desc = "A potent shield summoned by cultists to protect them while they prepare the final ritual"
+				N.icon_state = "shield-red"
+				N.health = 60
+				shields |= N
+		else
+			user << "<span class='warning'>Nar-Sie does not wish to be summoned!</span>"
```
Comment (Shadowlight213) [inline code/game/gamemodes/cult/ritual.dm]: Why? This prevents cult made by create antags summoning narsie. Limiting things to gamemodes is dumb. Were there actual issues with narsie being summoned outside of cult?

**Comment by ChangelingRain** [comment]
Also, checking if the target is sacced shouldn't be an absolute pain in the ass; right now, the only way to know for sure if the target is sacced is trying to use the summon nar-sie rune with 9 people, and that's just absolute crap to deal with.

**Comment by Shadowlight213** [comment]
True. It makes sense I suppose. With the rune announcment, it's riskier to check if narsie is summoned.

**Comment by ChangelingRain** [comment]
Robustin added a confirm prompt, so there's no actual risk to checking. I'd give the cult a 'status' display of some sort to check objectives and shit but that'd be a feature.

**Comment by Shadowlight213** [comment]
@ChangelingRain you'd need to add it to all other modes as well in that case.

**Pull request merged**